server/notification-providers/dingding.js

```diff
@@ -80,23 +80,6 @@ class DingDing extends NotificationProvider {
             .update(Buffer.from(`${timestamp}\n${secretKey}`, "utf8"))
             .digest("base64");
     }
-
-    /**
-     * Convert status constant to string
-     * @param {const} status The status constant
-     * @returns {string} Status
-     */
-    statusToString(status) {
-        // TODO: Move to notification-provider.js to avoid repetition in classes
```

Comment:
repeating ourselves twice out of 100 notification providers is not bad. Lets revert this change and remove the TODO.

Reply:
Yes, that's why I have not moved it to Notification provider base class.

I have moved it to utils so we only have it once now. And it is more like a utility function rather than being a method of any specific class.

Sorry if I'm not able to interpret your comment correctly. Please correct me if I'm wrong

Reply:
I meant that you should move this back to the individual notification providers where it came from and remove the TODO.

Reply:
Ok got it, reverted the changes and removed the TODO comment. Can you please check now